### PR TITLE
cli node support

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

File: src/katello/client/api/node.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+                'node': {
+                   'environment_ids': environment_ids
+                }
+              }
+        path = "/api/nodes/%s" % node_id
+        return self.server.PUT(path, data)[1]
+
+    def sync(self, node_id, env_id=None):
+        data = {}
+        if env_id != None:
```
Comment (daviddavis) [inline src/katello/client/api/node.py]: I believe you should use `env_id is not None` as None is an identity and `is` is used to compare identities while `==` is used to do equality comparisons.

From PEP 8:

> Comparisons to singletons like None should always be done with is or is not, never the equality operators

File: src/katello/client/core/node.py [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+
+    def run(self):
+        node_id = self.get_option("id")
+        node_name = self.get_option("name")
+
+        org_name = self.get_option('org')
+        env_name = self.get_option('env_name')
+        env_id = None
+        message = _("Syncing node, please wait... ")
+        if env_name != None:
```
Comment (daviddavis) [inline src/katello/client/core/node.py]: Same here.

File: test/katello/tests/core/node/node_data.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
+
+
+
```
Comment (daviddavis) [inline test/katello/tests/core/node/node_data.py]: 3 blank lines?

**Comment by daviddavis** [comment]
ACK pending small comments

**Comment by jlsherrill** [comment]
good catch @daviddavis

**Comment by daviddavis** [comment]
ack

**Pull request merged**